src/script/tracking/Countly.helpers.ts

```diff
@@ -35,10 +35,10 @@ export const getForcedErrorReportingStatus = (): boolean => forceActivateErrorRe
 
 // Init the forced activation of error reporting based on the environment
 export const initForcedErrorReporting = () => {
-  const {COUNTLY_ENABLE_LOGGING} = Config.getConfig();
+  const {COUNTLY_ENABLE_LOGGING, DISABLE_ENFORCE_COUNTLY} = Config.getConfig();
   const {isDev, isEdge, isInternal, isLocalhost, isStaging, name} = getWebEnvironment();
 
-  if (isDev || isEdge || isInternal || isLocalhost || isStaging) {
+  if (!DISABLE_ENFORCE_COUNTLY && (isDev || isEdge || isInternal || isLocalhost || isStaging)) {
```

Comment:
I feel like this would achieve the same thing:
```suggestion
  if (isEdge || isInternal) {
```

We only want to enforce it on internal envs, not really dev envs